src/main/java/frc/robot/hardware/interfaces/Iimu.java

```diff
@@ -2,7 +2,7 @@
 
 import edu.wpi.first.math.geometry.Rotation2d;
 
-public interface IGyro extends IDevice {
+public interface Iimu extends IDevice {
```

Comment:
```suggestion
public interface IIMU extends IDevice {
```
it's really silly but that's the correct way to follow the convention